src/models/captures.js

```diff
@@ -149,51 +155,35 @@ const trees = {
         },
       };
 
-      const query = `${
-        process.env.REACT_APP_API_ROOT
-      }/api/${getOrganization()}trees?filter=${JSON.stringify(lbFilter)}`;
-
-      const response = await Axios.get(query, {
-        headers: {
-          'content-type': 'application/json',
-          Authorization: session.token,
-        },
-      });
-      this.getTrees(response.data, {
+      const paramString = `filter=${JSON.stringify(lbFilter)}`;
+      const response = await this.queryCapturesApi({ paramString });
+      this.getCaptures(response.data, {
         page,
         rowsPerPage,
         orderBy,
         order,
         filter,
       });
     },
-    async getTreeAsync(id) {
-      const query = `${
-        process.env.REACT_APP_API_ROOT
-      }/api/${getOrganization()}trees/${id}`;
-      Axios.get(query, {
-        headers: {
-          'content-type': 'application/json',
-          Authorization: session.token,
-        },
-      })
+    async getCaptureAsync(id) {
+      this.queryCapturesApi({ id })
         .then((res) => {
-          this.getTree(res.data);
+          this.getCapture(res.data);
         })
         .catch((err) =>
           console.error(`ERROR: FAILED TO GET SELECTED TREE ${err}`),
         );
     },
     async getLocationName(payload, rootState) {
       if (
-        (rootState.trees.byId[payload.id] &&
-          rootState.trees.byId[payload.id].location &&
-          rootState.trees.byId[payload.id].location.lat !== payload.lat &&
-          rootState.trees.byId[payload.id].location.lon !== payload.lon) ||
-        !rootState.trees.byId[payload.id] ||
-        !rootState.trees.byId[payload.id].location
+        (rootState.captures.byId[payload.id] &&
+          rootState.captures.byId[payload.id].location &&
+          rootState.captures.byId[payload.id].location.lat !== payload.lat &&
+          rootState.captures.byId[payload.id].location.lon !== payload.lon) ||
+        !rootState.captures.byId[payload.id] ||
+        !rootState.captures.byId[payload.id].location
       ) {
-        const query = `https://nominatim.openstreetmap.org/reverse?format=jsonv2&lat=${payload.latitude}&lon=${payload.longitude}`;
+        const query = `https://nominatim.openscapturetmap.org/reverse?format=jsonv2&lat=${payload.latitude}&lon=${payload.longitude}`;
```

Comment:
@nmcharlton Seems like this may have replaced the 'tree' in 'street' with 'capture'

Reply:
Great spot!